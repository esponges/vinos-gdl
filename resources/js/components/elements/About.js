import React from 'react';

const About = () => {
    return (
        <div className="container">
            <h1>
                Información sobre envíos
            </h1>
            <ul>
                <li>

                </li>
            </ul>
        </div>
    );
}

