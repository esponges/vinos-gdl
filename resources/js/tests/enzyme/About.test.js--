import React from "react";
import About from "../../components/elements/About";
import { configure, mount, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

describe('about', () => {

    // beforeEach(() => {
    //     const wrapper = shallow(<About />)
    // });

    it('renders', () => {
        const wrapper = shallow(<About />);

        expect(toJson(wrapper)).toMatchSnapshot();
    })
});
