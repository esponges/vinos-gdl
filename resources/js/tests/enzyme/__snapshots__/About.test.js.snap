// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about renders 1`] = `
<div
  className="container"
  style={
    Object {
      "marginTop": "13%",
    }
  }
>
  <ul
    id="about-ul"
  >
    <h3
      className="mt-4"
    >
      ¿Quiénes somos?
    </h3>
    <li>
      ¡Hola! 
      <br />
       Somos Fer y Tomás. Somos millenials y treintones (¿ups?). Nos encanta pasar momentos divertidos con la nuestra familia y con nuestros amigos acompañados de un buen vino. Conocemos muy bien la industria de vinos y licores, por lo que  
      <b>
        puedes estar tranquilo; te vamos a atender muy bien.
      </b>
    </li>
    <h3
      className="mt-4"
    >
      ¿Qué buscamos?
    </h3>
    <li>
      Que no pagues sobreprecio en el super, tienda de conveniencia, vianata o app. Prepara con un día de anticipación tu consumo y te podemos ayudar a ahorrar desde la comodidad de tu casa.
      <em>
        Olvídate de estar cazando ofertas
      </em>
      . 
      <br />
      <b>
        Con nosotros siempre hay precios bajos.
      </b>
    </li>
    <h3
      className="mt-4 mb-3"
    >
      Nuestro compromiso
    </h3>
    <li>
      <b>
        Queremos que el vino sea la menor de tus preocupaciones
      </b>
        para ese viaje o evento en puerta, mantener bien llena la cava de tu hogar, y, en general, hacer tus compras de vino a domicilio más
       
      <em>
        baratas sin dejar de ser confiables.
      </em>
    </li>
  </ul>
  <h5
    className="mb-3"
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Object {
          "icon": Array [
            512,
            512,
            Array [],
            "f05a",
            "M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z",
          ],
          "iconName": "info-circle",
          "prefix": "fas",
        }
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
    ¿Tienes dudas? Contáctanos a
     
    <a
      href="mailto: hola@vinoreo.mx"
    >
      hola@vinoreo.mx
    </a>
     o directo a nuestro Whatsapp/DM.
  </h5>
  <Link
    to="/"
  >
    <Button
      active={false}
      disabled={false}
      variant="primary"
    >
      Regresar
    </Button>
  </Link>
</div>
`;
